README.md

```diff
@@ -30,7 +30,8 @@ Use the binaries from [the release page](https://github.com/snyk-tech-services/j
     --jiraProjectID=<12345>                                              // Jira project ID the tickets will be opened against
     --jiraProjectKey=<KEY>                                               // Jira project Key the tickets will be opened against
     --jiraTicketType=<Task|Bug|....>                                     // Optional. Type of ticket to open. Defaults to Bug. Please see the 'Notes' section below.
-    --severity=<critical|high|medium|low>                                // Optional. Severity threshold to open tickets for. Defaults to low.
+    --severityThreshold=<critical|high|medium|low>                       // Optional. Severity threshold to open tickets for. Defaults to low.
```

Comment:
Please don't change the option name. This tool is used in our customer's pipeline and changing this will break it.